common/models/user.js

```diff
@@ -869,9 +1003,21 @@ module.exports = function(User) {
     );
 
     UserModel.remoteMethod(
+      'verify',
+      {
+        description: 'Trigger user\'s identity verification.',
+        accepts: [
+          {arg: 'uid', type: 'string', required: true, http: {source: 'form'}},
```

Comment:
I am proposing to map this argument to a path-parameter, see my earlier https://github.com/strongloop/loopback/pull/3314/files#r110362900

What are the arguments for sending `uid` in the request body?